code/languages/Flint/models/Flint.behavior.mps

```diff
@@ -31935,5 +31935,96 @@
     <node concept="2tJIrI" id="70Gl2CcC2Pq" role="jymVt" />
     <node concept="3Tm1VV" id="70Gl2CcC2tr" role="1B3o_S" />
   </node>
+  <node concept="13h7C7" id="78uEmvU4O4G">
```

Comment:
![image](https://user-images.githubusercontent.com/15971736/179522830-ccf02949-2ecc-4d96-89ee-40f856583c35.png)
Comment and code does not match i don't think xor needs to be limited to 2